src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -358,12 +384,38 @@ waila.appliedenergistics2.Unlocked=Entsperrt
 waila.appliedenergistics2.Showing=Zeigt
 waila.appliedenergistics2.Contains=Enthält
 waila.appliedenergistics2.Channels=%1$d von %2$d Kanälen
+waila.appliedenergistics2.P2PUnlinked=Link getrennt
+waila.appliedenergistics2.P2PInputOneOutput=Verbunden (Eingangsseite)
+waila.appliedenergistics2.P2PInputManyOutputs=Verbunden (Eingangsseite) - %d Ausgänge
+waila.appliedenergistics2.P2POutput=Verbunden (Ausgangsseite)
+ 
+// TheOneProbe
+theoneprobe.appliedenergistics2.crafting=Herstellen: %1$s
+theoneprobe.appliedenergistics2.device_online=Gerät verfügbar
+theoneprobe.appliedenergistics2.device_offline=Gerät nicht verfügbar
+theoneprobe.appliedenergistics2.device_missing_channel=Gerät fehlt Kanal
+theoneprobe.appliedenergistics2.locked=Gesperrt
+theoneprobe.appliedenergistics2.unlocked=Entsperrt
+theoneprobe.appliedenergistics2.showing=Zeigt
+theoneprobe.appliedenergistics2.contains=Enthält
+theoneprobe.appliedenergistics2.channels=%1$d von %2$d Kanälen
+theoneprobe.appliedenergistics2.p2p_unlinked=Link getrennt
+theoneprobe.appliedenergistics2.p2p_input_one_output=Verbunden (Eingangsseite)
+theoneprobe.appliedenergistics2.p2p_input_many_outputs=Verbunden (Eingangsseite) - %d Ausgänge
+theoneprobe.appliedenergistics2.p2p_output=Verbunden (Ausgangsseite)
+theoneprobe.appliedenergistics2.p2p_frequency=Frequenz: %1$s
+theoneprobe.appliedenergistics2.stored_energy=%1$d / %2$d
 
 // Items
 item.appliedenergistics2.storage_cell_1k.name=1k-ME-Speicherzelle
 item.appliedenergistics2.storage_cell_4k.name=4k-ME-Speicherzelle
 item.appliedenergistics2.storage_cell_16k.name=16k-ME-Speicherzelle
 item.appliedenergistics2.storage_cell_64k.name=64k-ME-Speicherzelle
+item.appliedenergistics2.fluid_storage_cell_1k.name=1k-ME-Flüssigkeitenzelle
+item.appliedenergistics2.fluid_storage_cell_4k.name=4k-ME-Flüssigkeitenzelle
+item.appliedenergistics2.fluid_storage_cell_16k.name=16k-ME-Flüssigkeitenzelle
+item.appliedenergistics2.fluid_storage_cell_64k.name=64k-ME-Flüssigkeitenzelle
```

Comment:
To make the translation a bit more clear, we should say "Flüssigkeitsspeicherzelle" instead of "Flüssigkeitenzelle".

Reply:
```suggestion
item.appliedenergistics2.fluid_storage_cell_1k.name=1k-ME-Flüssigkeitsspeicherzelle
item.appliedenergistics2.fluid_storage_cell_4k.name=4k-ME-Flüssigkeitsspeicherzelle
item.appliedenergistics2.fluid_storage_cell_16k.name=16k-ME-Flüssigkeitsspeicherzelle
item.appliedenergistics2.fluid_storage_cell_64k.name=64k-ME-Flüssigkeitsspeicherzelle
```